Module.php

```diff
@@ -189,6 +221,156 @@ public function handleItemApiHydratePre(Event $event)
         $request->setContent($data);
     }
 
+    public function addUserFormElement(Event $event): void
+    {
+        /** @var \Omeka\Form\UserForm $form */
+        $form = $event->getTarget();
+        if ($form->getOption('is_public')) {
```

Comment:
Why check for is_public here?

Reply:
I use the omeka user form in module Guest to allow end users to set some options like locale, so it avoids to display this option in the public side.

Reply:
I'm not thrilled with the idea that PRs contain code that's specific to other modules. Is there a way for the Guest module to white list elements instead?